tests/Unit/UtilsTest.php

```diff
@@ -4,45 +4,92 @@
 use Automattic\LegacyRedirector\Utils;
 use Brain\Monkey;
 use Yoast\WPTestUtils\BrainMonkey\TestCase;
+use Automattic\LegacyRedirector\Tests\Unit\MonkeyStubs;
 
 /**
  * Capability Class Unit Test
  */
 final class UtilsTest extends TestCase {
 
 	/**
-	 * Utils::mb_parse_url test method
+	 * Setup any mocks before tests.
 	 *
-	 * @covers \Automattic\LegacyRedirector\Utils::mb_parse_url
+	 * @before
 	 */
-	public function test_mb_parse_url() {
-
-		Monkey\Functions\stubs(
-			array(
-				'wp_parse_url' => static function ( $url, $component ) {
-					return parse_url( $url, $component );
-				},
-			)
-		);
-
-		$url = 'https://www.example.org';
-		$this->do_assertion_mb_parse_url( $url, 'https', 'www.example.org', '', '' );
-
-		$url = 'https://www.example.org/';
-		$this->do_assertion_mb_parse_url( $url, 'https', 'www.example.org', '/', '' );
+	public static function initialSetup() {
```

Comment:
```suggestion
	public static function set_up() {
```

Try to get a bit of consistency with the usual naming.